rpc/legacyrpc/methods.go

```diff
@@ -124,7 +124,7 @@ var rpcHandlers = map[string]struct {
 	"getwalletfee":            {handler: getWalletFee},
 	"help":                    {handler: helpNoChainRPC, handlerWithChain: helpWithChainRPC},
 	"importprivkey":           {handler: importPrivKey},
-	"importscript":            {handlerWithChain: importScript},
+	"importscript":            {handler: importScript},
```

Comment:
The wallet struct type will eventually lose the rpc client field.  Instead there will be a separate package responsible for synchronizing the wallet.  I'd rather this handler continue to require the rpc client instead and use it to begin the rescan.